src/server.c

```diff
@@ -3731,6 +3737,13 @@ void call(client *c, int flags) {
             server.lua_caller->flags |= CLIENT_FORCE_AOF;
     }
 
+    /* Some commands may contain sensitive data that should
+     * not be available in the slowlog. */
+    if (c->flags & CLIENT_PREVENT_LOGGING) {
+        c->flags &= ~CLIENT_PREVENT_LOGGING;
+        flags &= ~CMD_CALL_SLOWLOG;
```

Comment:
https://youtu.be/o7v0PePgVIs?t=9
in which case, i wonder how the tests passed?

Reply:
How were they passing O_O

Reply:
ohh, sorry, for some reason i thought CMD_CALL_SLOWLOG means not to log it, so i though we want to turn on the flag.